src/modules/layout/components/__tests__/ToolbarItem.spec.js

```diff
@@ -9,14 +9,21 @@ import ReactTooltip from 'react-tooltip'
 import { ThemeProvider } from 'styled-components'
 import theme from '../../../theme/constants/theme'
 
+import { Provider } from 'react-redux'
+
+import createReduxStore from '../../../app/createReduxStore'
+
 describe('ToolbarItem', () => {
   it('should rebuild tooltip items on mount', () => {
+    const store = createReduxStore()
     const original = ReactTooltip.rebuild
     ReactTooltip.rebuild = jest.fn()
 
     mount(
       <ThemeProvider theme={theme}>
-        <ToolbarItem href='http://example.com' icon={faSmile} text='Click here!' />
+        <Provider store={store}>
```

Comment:
Toolbar item should probably not be connected to the state. Therefore this can be removed

Reply:
removed